src/txmempool.cpp

```diff
@@ -367,7 +367,7 @@ void CTxMemPool::check(const CCoinsViewCache *pcoins) const
     if (nCheckFrequency == 0)
         return;
 
-    if (insecure_rand() >= nCheckFrequency)
+    if (GetRand(std::numeric_limits<uint32_t>::max()) >= nCheckFrequency)
```

Comment:
This is technically a semantic change because `insecure_rand()` can return `0xFFFFFFFF`, and `GetRand(std::numeric_limits<uint32_t>::max()` [can't](https://github.com/zcash/zcash/blob/0f091f228cdb1793a10ea59f82b7c7f0b93edb7a/src/random.cpp#L53).

It probably doesn't matter given the intended semantics of `nCheckFrequency`.